docs/en/plugins/server.md

```diff
@@ -61,4 +61,59 @@ server:
   #
   # Default: 60s
   relay_timeout: 60s
-```
\ No newline at end of file
+```
+
+### Server initialization
+
+The `on_init` section is used for application initialization or warming up before starting workers. It allows you to set
```

Comment:
You may also note, that:
1. User may run anything with this `on_init` command. Not only roadrunner php worker (like service).
2. If the command fails, RR will log that and continue execution. So, `on_init` fail (non 0 exit code) will not interrupt the execution.